services/api/src/routes/users.js

```diff
@@ -35,24 +35,14 @@ router
       data: expandRoles(authUser),
     };
   })
-  .patch(
-    '/me',
-    validateBody({
-      firstName: yd.string(),
-      lastName: yd.string(),
-      timeZone: yd.string(),
-      theme: yd.string(),
-    }),
-    async (ctx) => {
-      const { authUser } = ctx.state;
-      authUser.assign(ctx.request.body);
-
-      await authUser.save();
-      ctx.body = {
-        data: expandRoles(authUser),
-      };
-    }
-  )
+  .patch('/me', validateBody(User.getUpdateValidation()), async (ctx) => {
```

Comment:
there is a side effect when doing this, one have to keep track of keeping the docs in sync
before it was hardcoded, adding a new field in the user definition wouldn't require an update to the docs for patch /users/me endpoint 

anyway not a problem just something to think about.